docs/src/main/asciidoc/getting-started.adoc

```diff
@@ -1266,7 +1266,7 @@ Tests in error:
 ----
 ====
 
-This error occurs because you have a new contract from which a test was generated and it
+This error occurs because you have a new contract from which a test was generated, and it
```

Comment:
This one is correct. Note how "it failed since you have not implemented the feature" could be a sentence by itself.